TramsDataApi/Gateways/ConcernsCaseGateway.cs

```diff
@@ -30,13 +30,15 @@ public IList<ConcernsCase> GetConcernsCaseByTrustUkprn(string trustUkPrn, int pa
                 .Where(c => c.TrustUkprn == trustUkPrn)
                 .Skip((page - 1) * count)
                 .Take(count)
+                .Include(x => x.Decisions)
```

Comment:
If we follow this example, we'll need to include every action table (and possibly every related lookup table as well). I think that might be something we should do, but with care to make sure we're only using it when we really need all the returned data. Might be one to look at after decoupling when we have more scope to see how stuff is used

Reply:
It's one to come back to. But we shouldn't retrieve decisions without the concern, and it will be a mess to have to have methods that are GetConcern, and GetConcern with a xxx.
If performance becomes an issue it's something to look at and tackle in a few ways, but repos should return aggregates IMO.

Reply:
Agreed